src/NodaTime/Text/UnparsableValueException.cs

```diff
@@ -31,5 +31,14 @@ public UnparsableValueException(string message)
             : base(message)
         {
         }
+
+        /// <summary>
+        /// Creates a new UnparsableValueException with the given message and inner exception
+        /// </summary>
+        /// <param name="message">The failure message</param>
+        /// <param name="innerException">The inner exception</param>
+        public UnparsableValueException(string message, Exception innerException) : base(message, innerException)
```

Comment:
okay but why? In case someone else wants to use it in a different context, externally?